src/Directory.Build.props

```diff
@@ -95,6 +96,7 @@
 	</ItemGroup>
 
 	<PropertyGroup Condition="$(_IsCatalyst)">
+		<DefineConstants>$(DefineConstants);XAMARIN</DefineConstants>
```

Comment:
We should correct the props so that XAMARIN is only defined for actual Xamarin targets (ie not .net6-X targets)?